<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>821</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;</string>
    </property>
    <widget class="QLabel" name="labelLogin">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>811</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/child-girl-schoolgirl-elementary-school-student-123686003.jpg);
border-radius :20px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>771</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(177, 177, 177,75);
border-radius:20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>221</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>ĐĂNG NHẬP </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Tên đăng nhập : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineTenDangNhap">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border-radius:5px;
background-color: rgb(223, 223, 223,150 );
color: white;
	font: 75 18pt &quot;Tahoma&quot;;
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Mật khẩu : </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDangNhap">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>330</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(255, 255, 0);
}
QPushButton:hover{
background-color: rgb(255, 0, 0);
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>ĐĂNG NHẬP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMatKhau">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>210</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border-radius:5px;
background-color: rgb(223, 223, 223,150 );
color: white;
	font: 75 18pt &quot;Tahoma&quot;;
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="textError">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>280</y>
       <width>571</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../img/login.qrc"/>
 </resources>
 <connections/>
</ui>
